packages/react-theme/src/global/spacings.ts

```diff
@@ -0,0 +1,43 @@
+import type { HorizontalSpacingTokens, SpacingTokens, VerticalSpacingTokens } from '../types';
+
+export const spacings: SpacingTokens = {
```

Comment:
do these raw values need to be exported? is there a precedent for making them available?

Reply:
oh, i guess i'm assuming they are exported top level, which might not be the case